integration/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3.7'
+
+x-stas: &common-opi
+    image: ghcr.io/opiproject/opi-poc:main
```

Comment:
Have we pushed a container to GHCR yet?

Reply:
no, and we don;t have to, compose will build everything from dockerfiles